tsconfig.json

```diff
@@ -2,7 +2,8 @@
   "compilerOptions": {
     "target": "es2017",
     "lib": [
-      "esnext"
+      "esnext",
+      "dom"
```

Comment:
This should be removed.

Reply:
![chaneg](https://user-images.githubusercontent.com/26249903/96531147-97779600-12a6-11eb-8fd8-b9f9f64f3617.png)
This was added because of `msgpack` build:watch error

Reply:
If i don't add this i get this error

Reply:
Strange, it seams that `msgpack` has this dependency. OK. Feel free to merge.